src/table.ts

```diff
@@ -1640,6 +1576,40 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     makeNextBatchRequest();
   }
 
+  #readRowsReqOpts(
```

Comment:
nit that I'm surprised our linter didn't catch - I think you shouldn't use #private notation per our [ts style guide](https://google.github.io/styleguide/tsguide.html#class-members). Does this helper function for sure need to be private? (It might - I just want to know more about your thought process! 😁 )

Reply:
Additional question - does this need to stay within the table class, or could it be a helper function similar to `populateAttemptHeader` down below? Tell me more! 🙂

Reply:
I think doing what `populateAttemptHeader` is a good option. The con is that the function also needs to be able to accept table so needs a fourth argument. The pro is that we don't have to have a private ECMA script function that is hard to debug. It's a good suggestion overall that I was wondering about.

We definitely should hide this function from the user either way.